cloudferrylib/utils/log.py

```diff
@@ -61,7 +64,7 @@ def configure_logging():
         for handler in logger.handlers:
             if handler.name == 'console':
                 handler.setLevel(logging.DEBUG)
-    sys.stdout = StdoutLogger()
+    sys.stdout = STDOUT_LOGGER
```

Comment:
Let's make `sys.stdout` optional, it's very inconvenient to debug with stdout set to logger.